src/App.js

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
 
+let todoList = [
+  {
+    id: 1,
+    title: "Wake up"
+  },
+  {
+    id: 2,
+    title: "Take a shower"
+  },
+  {
+    id: 3,
+    title: "Take a coffee"
+  },
+  {
+    id: 4,
+    title: "Learn React"
+  }
+];
+
 function App() {
-  return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+
+  return(
+    <span>
+      <h1>ToDo List</h1>
+      <ul>
+        {todoList.map( (item) =>
+          <li key={item.id}>{item.title}</li>
+        )}
+      </ul>
+    </span>
```

Comment:
In general this all looks really good except for the parent element being a `span`. Typically if you just need an element to wrap or group other top-level elements you'll want to use something less descriptive, like a `div`. React also has something even better, called a [React fragment](https://react.dev/reference/react/Fragment) that looks like this `<>...</>` which works to group elements together but doesn't render an extra element to the DOM.

```suggestion
    <>
      <h1>ToDo List</h1>
      <ul>
        {todoList.map( (item) =>
          <li key={item.id}>{item.title}</li>
        )}
      </ul>
    </>
```